src/ts/jsoneditor.ts

```diff
@@ -29,6 +29,15 @@ document.addEventListener('DOMContentLoaded', () => {
     const JsonEditorHelperC = new JsonEditorHelper(entity);
     JsonEditorHelperC.init((about.page === 'edit' || about.page === 'template-edit'));
 
+    if (about.type == 'experiments_templates') {
+      const entityWithId = {
+        type: entity.type,
+        id: parseInt(about.id),
```

Comment:
parseInt needs to have the radix (10) as second argument

Reply:
I didn't know. I will add it.

Reply:
There are many occurrences in the ts code base without the radix

Reply:
I know, I'll change that.